pkg/runtime/helper.go

```diff
@@ -46,6 +46,59 @@ func UnsafeObjectConvertor(scheme *Scheme) ObjectConvertor {
 	return unsafeObjectConvertor{scheme}
 }
 
+// PreferredGroupVersion returns the desired group and version reported by the GroupVersioner, or
+// false if no such group version was indicated.
+func PreferredGroupVersion(target GroupVersioner) (unversioned.GroupVersion, bool) {
+	if group, ok := target.PrefersGroup(); ok {
```

Comment:
This construct is reading weird for me.  How are you choosing a group without the `GroupKind`?